email_template_qweb/models/mail_template.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models, tools
+
+
+class MailTemplate(models.Model):
+    _inherit = "mail.template"
+
+    body_type = fields.Selection(
+        [("qweb", "QWeb"), ("qweb_view", "QWeb View")],
+        "Body templating engine",
+        default="qweb",
+        required=True,
+    )
+    body_view_id = fields.Many2one("ir.ui.view", domain=[("type", "=", "qweb_view")])
```

Comment:
We should use `qweb` because `qweb_view` does not exist in the list of view types
```suggestion
    body_view_id = fields.Many2one("ir.ui.view", domain=[("type", "=", "qweb")])
```

Reply:
@dzungtran89 I've done the changes as per your feedback, Please review it.

Reply:
Thank you, btw you might want to remove the file `email_template_qweb/i18n/en.po`, otherwise the `pre-commit` will fail

Reply:
Sure, I'll remove it

Reply:
Hello @dzungtran89, I've removed the `email_template_qweb/i18n/en.po`. Please review it.
Thank you